README.md

```diff
@@ -46,353 +54,13 @@ ACH is under active development but already in production for multiple companies
 	* Addenda Type Code 18 (IAT Optional)
 	* Addenda Type Code 98 (NOC)
 	* Addenda Type Code 99 (Return)
+-->
 
-## Project Roadmap
-* Additional SEC codes will be added based on library users needs. Please open an issue with a valid test file.
-* Review the project issues for more detailed information
+## HTTP API
```

Comment:
Should we mention more here? Include a swagger doc just for this project? Refer people to the future `api.moov.io` service?